src/test/app/Offer_test.cpp

```diff
@@ -4519,74 +4517,117 @@ class Offer_test : public beast::unit_test::suite
+        testSelfCross(false, features);
+        testSelfCross(true, features);
+        testNegativeBalance(features);
+        testOfferCrossWithXRP(true, features);
+        testOfferCrossWithXRP(false, features);
+        testOfferCrossWithLimitOverride(features);
+        testOfferAcceptThenCancel(features);
+        testOfferCancelPastAndFuture(features);
+        testCurrencyConversionEntire(features);
+        testCurrencyConversionIntoDebt(features);
+        testCurrencyConversionInParts(features);
+        testCrossCurrencyStartXRP(features);
+        testCrossCurrencyEndXRP(features);
+        testCrossCurrencyBridged(features);
+        testOfferFeesConsumeFunds(features);
+        testOfferCreateThenCross(features);
+        testSellFlagBasic(features);
+        testSellFlagExceedLimit(features);
+        testGatewayCrossCurrency(features);
+        testPartialCross (features);
+        testXRPDirectCross (features);
+        testDirectCross (features);
+        testBridgedCross (features);
+        testSellOffer (features);
+        testSellWithFillOrKill (features);
+        testTransferRateOffer(features);
+        testSelfCrossOffer (features);
+        testSelfIssueOffer (features);
+        testBadPathAssert (features);
+        testDirectToDirectPath (features);
+        testSelfCrossLowQualityOffer (features);
+        testOfferInScaling (features);
+        testOfferInScalingWithXferRate (features);
+        testOfferThresholdWithReducedFunds (features);
+        testTinyOffer (features);
+        testSelfPayXferFeeOffer (features);
+        testSelfPayUnlimitedFunds (features);
+        testRequireAuth (features);
+        testMissingAuth (features);
+        testRCSmoketest (features);
+        testSelfAuth (features);
+        testTickSize (features | featureTickSize);
+    }
+
+    void run () override
+    {
+        using namespace jtx;
+        auto const all           = supported_amendments();
+        FeatureBitset const flow{featureFlow};
+        FeatureBitset const f1373{fix1373};
+        FeatureBitset const flowCross{featureFlowCross};
+        (void) flow;
+
+        // The first three test variants below passed at one time in the past
+        // (and should still pass) but are commented out to conserve test time.
+//      testAll(all - flow - f1373 - flowCross);
+//      testAll(all - flow - f1373            );
+//      testAll(all        - f1373 - flowCross);
+        testAll(all        - f1373            );
+        testAll(all                - flowCross);
+        testAll(all                           );
+    }
+};
+
+class Offer_manual_test : public Offer_test
+{
+    void run() override
+    {
+        using namespace jtx;
+        auto const all           = supported_amendments();
+        auto const& feeEscalation{featureFeeEscalation};
+        auto const& flow{featureFlow};
+        auto const& f1373{fix1373};
+        auto const& flowCross{featureFlowCross};
+        auto const& f1513{fix1513};
```

Comment:
These would benefit from being declared as `FeatureBitset const` as well.  And, while you're in there, the many spaces following the declaration of local `all` on lines 4579 and 4601 no longer align with anything.  We might as well lose those spaces.

Reply:
Fixed